builder/builder.mk

```diff
@@ -333,10 +333,10 @@ push-dev: docker/$(LCNAME).docker.tag.local
 			exit 1 ;\
 		fi; \
 		if [[ '$(VERSION)' != *-* ]]; then \
-			printf "$(RED)push-dev: BUILD_VERSION $(BUILD_VERSION) is not a dev version$(END)\n" >&2 ;\
+			printf "$(RED)push-dev: BUILD_VERSION $(patsubst v%,%,$(VERSION)) is not a dev version$(END)\n" >&2 ;\
```

Comment:
Could this just be

```
VERSION_NO_V=$(patsubst v%,%,$(VERSION))
```

and then use $(VERSION_NO_V) everywhere instead of repeating the `patsubst`?

Reply:
Hmm, the "begone with BUILD_VERSION" commit should have changed this to
```
printf "$(RED)push-dev: VERSION $(VERSION) is not a dev version$(END)\n" >&2 ;\
```

Reply:
Yeah, but did _all_ those `patsubst` calls go away in `begone with BUILD_VERSION`?

Reply:
Most of them did.  And the ones that remain should be painful and verbose, because they are indicators of remaining inconsistency in the build system.

Reply:
👍